rllib/algorithms/tests/test_callbacks.py

```diff
@@ -176,7 +176,7 @@ def test_on_episode_created(self):
                 RandomEnv,
                 env_config={
                     "max_episode_len": 200,
-                    "p_done": 0.0,
+                    "p_terminated": 0.0,
```

Comment:
Changed this in RandomEnv:
`p_terminated`: change of the episode to get `terminated=True`
`max_episode_len`: after this many ts, we return `truncated=True`